homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -132,19 +127,19 @@ async def async_handle_start_charge(service: ServiceCall) -> None:
             # for the charging request to reach the car.
             result = await hass.async_add_executor_job(data_store.leaf.start_charging)
             if result:
-                _LOGGER.debug("Start charging sent, request updated data in 1 minute")
-                check_charge_at = utcnow() + timedelta(minutes=1)
+                LOGGER.debug("Start charging sent, request updated data in 1 minute")
+                check_charge_at = utcnow() + MIN_UPDATE_INTERVAL
```

Comment:
This is a different interval then the old one, why?

Reply:
Hi @eavanvalkenburg,   Thanks for the review. We shouldn't request updates from the Nissan servers more frequently than `MIN_UPDATE_INTERVAL`, so I changed the hardcoded 1 minute to use the `MIN_UPDATE_INTERVAL` const (which is currently two mins).    The debug log message needs updating though. Fixing...